src/sdk/object_sdk.js

```diff
@@ -642,7 +642,7 @@ class ObjectSDK {
             params.content_type = source_md.content_type;
         }
         try {
-            if (params.xattr) params.xattr = _.omitBy(params.xattr, (val, name) => name.startsWith('noobaa-namespace'));
+            if (params.xattr) params.xattr = _.omitBy(params.xattr, (val, name) => name.startsWith?.('noobaa-namespace'));
```

Comment:
this happens when the name is SYMBOL  and not a string right?

Reply:
right